packages/@lwc/engine-core/src/shared/format.ts

```diff
@@ -38,3 +45,9 @@ export function getErrorComponentStack(vm: VM): string {
 
     return wcStack.reverse().join('\n\t');
 }
+
+export function addErrorComponentStack(vm: VM, error: any): void {
+    if (!isFrozen(error) && isUndefined(error.wcStack)) {
+        error.wcStack = getErrorComponentStack(vm);
```

Comment:
Side question: Do we want to make the `wcStack` property enumerable? For example, the `stack` property isn't.

Reply:
I made it non-enumerable.